doc/decisions/0_drafts/transformations.md

```diff
@@ -176,10 +176,10 @@ A drawback to this solution is that it adds some complexity to `libelektra-core`
 
 ## Related Decisions
 
-- [Hooks](../4_partially_implemented/hooks.md)
```

Comment:
Weird that this does not conflict, even though the commit SHA is different than on master. 👀

Reply:
I expected a conflict as well, but seems git is smart enough to ignore it since the file is identical on both sides.